src/components/Comments/Comment.js

```diff
@@ -215,6 +289,7 @@ const Comment = ({
       </View>
 
       <BlockComponent ref={refBlockComponent} screen={'feed_comment_item'} />
+      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} height={182} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `BottomSheetMenu` component is well implemented. This will allow users to interact with the direct messaging options from a bottom sheet menu.


<!-- This is an auto-generated comment by OSS CodeRabbit -->